projects/cli-files/go-ls/cmd/root.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+)
+
+func Execute() error {
+	pathArr := os.Args[1:]
+
+	var path string
+	var comma bool
+
+	if len(pathArr) == 0 {
+		path = "."
+		directoryOutput(path, comma)
+		return nil
+	}
+
+	if pathArr[0] == "-m" {
+		comma = true
+	} else {
+		comma = false
+	}
+
+	if len(pathArr) == 1 && comma {
+		path = "."
+		directoryOutput(path, comma)
+		return nil
+	}
+
+	var i int
+	if comma {
+		i = 1
+	} else {
+		i = 0
+	}
+
+	for i < len(pathArr) {
+
+		isPathValid, err := os.Stat(pathArr[i])
+		if err != nil {
+			return fmt.Errorf("%s no such file or directory", pathArr[i])
+		}
+
+		isDirectory := isPathValid.IsDir()
+		if !isDirectory {
+			if comma && i < len(pathArr)-1 {
+				os.Stdout.Write([]byte(pathArr[i] + ", "))
+			} else {
+				os.Stdout.Write([]byte(pathArr[i] + "  "))
+			}
```

Comment:
I'd generally aim to factor out what's common here, so probably would write either:
```suggestion
			os.Stdout.Write([]byte(pathArr[i]))
			if comma && i < len(pathArr)-1 {
				os.Stdout.Write(b", ")
			} else {
				os.Stdout.Write(b"  ")
			}
```

or
```suggestion
			var suffix string
			if comma && i < len(pathArr)-1 {
				suffix = ", "
			} else {
				suffix = "  "
			}
			os.Stdout.Write([]byte(pathArr[i] + suffix))
```

This makes it more clear what's the same about both cases (in both cases you're printing the filename), and what's different (the suffix you're printing after).